radar_object_data obj 
uint8 obj_id
